.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+  "presets": [["env", {
+    "targets": {
+      "node": "6"
```

Comment:
Be careful here, people still use Node4, verdaccio v2.x still support Node 4. If you switch to Node6 (which is the minimum since last month. You need yo warn users (README or something), otherwise, they will get compiler errors on runtime.

Reply:
The 4.x release line is going end of life April 30th, I add engines to package.json and a hint to the README